src/mlpack/tests/one_hot_encoding_test.cpp

```diff
@@ -0,0 +1,166 @@
+/**
+ * @file one_hot_encoding_test.cpp
+ * @author Jeffin Sam
+ *
+ * Tests for onehotencoding().
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <sstream>
```

Comment:
Not sure I see the need for the include, but maybe I missed something?

Reply:
Removed